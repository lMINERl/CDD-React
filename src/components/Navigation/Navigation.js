import React from 'react';
import Profile from '../NavigationProfile';
import Search from '../Search';
import { ButtonToolTipNotifcation } from '../../stories/Button.stories';
import logo from '../../assets/logo.svg';
const Navigation = props => {
  return (
    <div className="navigation">
      <div className="navigation navigation__logo navigation--dark">
        <img src={logo} alt="logo"></img>
      </div>
      <div className="navigation navigation__menu navigation--standard w-75">
        {/* input */}
        <Search />
        {/* button rounded */}
        <div className="mr-auto ml-auto">
          {ButtonToolTipNotifcation(
            <i className="fa fa-shopping-bag"></i>,
            'button--orange button--rounded',
            'Shopping crat',
            'dark',
            235
          )}
        </div>
        {/* profile */}
        <Profile />
      </div>
    </div>
  );
};

export default Navigation;
