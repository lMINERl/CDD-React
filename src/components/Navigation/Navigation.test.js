import { React, shallow } from '../../test/Configurations';
import Navigation from './Navigation';

describe('<Navigation />', () => {
  test('renders', () => {
    const wrapper = shallow(<Navigation />);
    expect(wrapper).toMatchSnapshot();
  });
});
