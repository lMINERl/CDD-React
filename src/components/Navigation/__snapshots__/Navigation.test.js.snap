// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navigation /> renders 1`] = `
<div
  className="navigation"
>
  <div
    className="navigation navigation__logo navigation--dark"
  >
    <img
      alt="logo"
      src="logo.svg"
    />
  </div>
  <div
    className="navigation navigation__menu navigation--standard w-75"
  >
    <Search />
    <div
      className="mr-auto ml-auto"
    >
      <div
        className="button button--orange button--rounded tooltip-informed"
      >
        <div
          className="tooltip-informed__text tooltip-informed__text--bright"
        >
          Shopping crat
        </div>
        <div
          className="button__notify"
        >
          235
        </div>
        <i
          className="fa fa-shopping-bag"
        />
      </div>
    </div>
    <NavigationProfile />
  </div>
</div>
`;
